reporter/html.js

```diff
@@ -723,6 +723,20 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	// Only add this information if stackTrace is supported
+	if ( Error().stack ) {
```

Comment:
simplifying sourceFromStacktrace would cause a backward compatibility issue on `details.source`. It would be better to do this for 2.0